php-practice/section4.php

```diff
@@ -85,35 +84,36 @@ public function playCampaign() {
  * QUESTION 1
  *
  * What are the 2 main things that you can create inside of a class?
- * ANSWER:
+ * ANSWER: property variables and the constructor
```

Comment:
Good. I was actually looking for:

1. Properties (i.e. "class variables")
2. Methods (i.e. "class functions")

The constructor fits under 2.